src/_pytest/terminal.py

```diff
@@ -112,6 +113,21 @@ def __call__(
         namespace.quiet = getattr(namespace, "quiet", 0) + 1
 
 
+class TestStatus(NamedTuple):
+    """Used to store the test status result category, shortletter and verbose word.
+    For example ``"rerun", "R", ("RERUN", {"yellow": True})``.
+
+    category: the class of result, for example “passed”, “skipped”, “error”, or the empty string
+    letter: the shortletter shown as testing progresses, for example ".", "s", "E", or the empty string.
+    word: verbose word is shown as testing progresses in verbose mode, for example "PASSED", "SKIPPED",
+    "ERROR", or the empty string.
+    """
```

Comment:
This is based on what we have in the docstring for the hook over [here](https://github.com/pytest-dev/pytest/blob/23e343af6018ef3dcf8650e9b3c4a8da0fabe566/src/_pytest/hookspec.py#L813-L824) in case that's helpful information ℹ️